### PR TITLE
(SIMP-4989) Update packages.yaml for Beta2

### PR DESCRIPTION
**Pull request by brandonrdn**
Updated the CentOS 7 packages.yaml for the following:
  - postgresql
  - clamav
  - vendored-r10k
  - elasticsearch
  - logstash
  - rubygem-rake-compiler
Updated all packages.yaml and the pkglist for CentOS 7 to remove
razor-server

### CODE REVIEW

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 1]
```diff
@@ -151,46 +151,46 @@ pdsh-rcmd-ssh:
...
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-server-9.6.4-1PGDG.rhel6.x86_64.rpm
+  :rpm_name: postgresql96-server-9.6.9-1PGDG.rhel6.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-server-9.6.9-1PGDG.rhel6.x86_64.rpm
 pssh:
   :rpm_name: pssh-2.3.1-5.el6.noarch.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/pssh-2.3.1-5.el6.noarch.rpm
 puppet-agent:
-  :rpm_name: puppet-agent-1.10.9-1.el6.x86_64.rpm
-  :source: http://yum.puppetlabs.com/el/6/PC1/x86_64/puppet-agent-1.10.9-1.el6.x86_64.rpm
+  :rpm_name: puppet-agent-1.10.10-1.el6.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: This is not the latest available (1.10.14-1).  How do we decide which version to use?

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 2]
```diff
@@ -223,13 +223,10 @@ qstat:
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/qstat-2.11-9.20080912svn311.el6.x86_64.rpm
 radiusclient-ng:
   :rpm_name: radiusclient-ng-0.5.6-5.el6.x86_64.rpm
-  :source: http://mirror.cogentco.com/pub/linux/epel/6/x86_64/radiusclient-ng-0.5.6-5.el6.x86_64.rpm
-razor-server:
-  :rpm_name: razor-server-1.6.1-1.el6.noarch.rpm
-  :source: https://yum.puppetlabs.com/el/6Server/PC1/x86_64/razor-server-1.6.1-1.el6.noarch.rpm
+  :source: http://lug.mtu.edu/epel/6/x86_64/Packages/r/radiusclient-ng-0.5.6-5.el6.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: python-six:  Why is this in 6_x_Alpha_Dependencies and not 6_X_Dependencies?  Does this mean we don't need it?

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 3]
```diff
@@ -278,40 +275,40 @@ simp-lastbind:
 simp-ppolicy-check-password:
   :rpm_name: simp-ppolicy-check-password-2.4.39-0.el6.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/simp-ppolicy-check-password-2.4.39-0.el6.x86_64.rpm
-simp-vendored-r10k-2.2.2-0.noarch.rpm:
+simp-vendored-r10k:
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: rubygem-rake-compiler:  should be rubygem-rake-compiler-0.9.3-1 ?  We have 0.9.3-2 in packagecloud, but at yum.puppetlabs.com, latest is 0.9.3-1.  Need to understand the difference?

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 4]
```diff
@@ -278,40 +275,40 @@ simp-lastbind:
...
   :rpm_name: simp-vendored-r10k-2.2.2-0.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: wrong version. Should be 2.6.2-1

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 5]
```diff
@@ -278,40 +275,40 @@ simp-lastbind:
...
   :source: https://packagecloud.io/simp-project/6_X/packages/el/6/simp-vendored-r10k-2.2.2-0.noarch.rpm
-simp-vendored-r10k-doc-2.2.2-0.noarch.rpm:
+simp-vendored-r10k-doc:
   :rpm_name: simp-vendored-r10k-doc-2.2.2-0.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: wrong version.  2.6.2-1

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml [hunk 6]
```diff
@@ -278,40 +275,40 @@ simp-lastbind:
...
-simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm:
+simp-vendored-r10k-gem-multipart-post:
   :rpm_name: simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm
   :source: https://packagecloud.io/simp-project/6_X/packages/el/6/simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm
-simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm:
+simp-vendored-r10k-gem-puppet_forge:
   :rpm_name: simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
   :source: https://packagecloud.io/simp-project/6_X/packages/el/6/simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
-simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm:
+simp-vendored-r10k-gem-r10k:
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: Should be 2.6.2-1

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 7]
```diff
@@ -6,50 +6,41 @@ chkrootkit:
   :rpm_name: chkrootkit-0.50-4el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/chkrootkit-0.50-4el7.x86_64.rpm
 clamav:
-  :rpm_name: clamav-0.99.2-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/c/clamav-0.99.2-1.el7.x86_64.rpm
+  :rpm_name: clamav-0.100.0-2.el7.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: For apr-util-ldap, had to go to vault.centos.org/7.4.1708/os/x86_64/Packages to find the RPM.  Source listed did not work.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 8]
```diff
@@ -6,50 +6,41 @@ chkrootkit:
...
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/c/clamd-0.100.0-2.el7.x86_64.rpm
 elasticsearch:
-  :rpm_name: elasticsearch-5.6.7.rpm
-  :source: https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.7.rpm
+  :rpm_name: elasticsearch-5.6.10.rpm
+  :source: https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.10.rpm
 elasticsearch-curator:
-  :rpm_name: elasticsearch-curator-5.4.1-1.x86_64.rpm
-  :source: https://packages.elastic.co/curator/5/centos/7/Packages/elasticsearch-curator-5.4.1-1.x86_64.rpm
+  :rpm_name: elasticsearch-curator-5.5.1-1.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Latest version is 5.5.4-1

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 9]
```diff
@@ -58,22 +49,22 @@ gweb:
...
 haveged:
   :rpm_name: haveged-1.9.1-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/h/haveged-1.9.1-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/h/haveged-1.9.1-1.el7.x86_64.rpm
 incron:
   :rpm_name: incron-0.5.10-8.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/i/incron-0.5.10-8.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/i/incron-0.5.10-8.el7.x86_64.rpm
 libarchive-devel:
   :rpm_name: libarchive-devel-3.1.2-10.el7_2.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: URL no longer works.  Looks like another for the vault.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 10]
```diff
@@ -96,17 +87,17 @@ pdsh-rcmd-ssh:
...
+  :rpm_name: postgresql96-libs-9.6.9-1PGDG.rhel7.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/postgresql96-libs-9.6.9-1PGDG.rhel7.x86_64.rpm
 postgresql96-server:
-  :rpm_name: postgresql96-server-9.6.4-1PGDG.rhel7.x86_64.rpm
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/postgresql96-server-9.6.4-1PGDG.rhel7.x86_64.rpm
+  :rpm_name: postgresql96-server-9.6.9-1PGDG.rhel7.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/postgresql96-server-9.6.9-1PGDG.rhel7.x86_64.rpm
 pssh:
   :rpm_name: pssh-2.3.1.SIMP-5.el7.noarch.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/pssh-2.3.1.SIMP-5.el7.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: - puppet-agent: 1.10.9-1 is not the latest and does not match CentOS6 packages.yaml
- puppet-client-tools:  I found at the same URL as puppet-agent (note no '7Server', simply '7' in the middle of URL)

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 11]
```diff
@@ -130,22 +121,19 @@ puppetserver:
...
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-linecache2-1.0.0-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-linecache2-1.0.0-1.el7.noarch.rpm
 python-redis:
   :rpm_name: python-redis-2.10.3-1.el7.noarch.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-redis-2.10.3-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-redis-2.10.3-1.el7.noarch.rpm
 python-simplejson:
   :rpm_name: python-simplejson-3.3.3-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/p/python-simplejson-3.3.3-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/p/python-simplejson-3.3.3-1.el7.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: I didn't see python-simplejson at that URL, but did see it in packagecloud in the 6_X_Dependencies repo

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 12]
```diff
@@ -202,55 +190,73 @@ rubygem-rake:
   :source: http://mirrors.advancedhosters.com/centos/7.4.1708/os/x86_64/Packages/rubygem-rake-0.9.6-33.el7_4.noarch.rpm
 rubygem-rake-compiler:
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: rubygem-rake:  Another package for the CentOS vault URL

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 13]
```diff
@@ -202,55 +190,73 @@ rubygem-rake:
...
   :rpm_name: rubygem-rake-compiler-0.9.3-2.el7.noarch.rpm
-  :source: http://yum.puppetlabs.com/el/7/dependencies/x86_64/rubygem-rake-compiler-0.9.3-2.el7.noarch.rpm
+  :source: https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/r/rubygem-rake-compiler-0.9.3-2.el7.noarch.rpm
 rubygem-stomp:
   :rpm_name: rubygem-stomp-1.3.5-1.el7.noarch.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/r/rubygem-stomp-1.3.5-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/r/rubygem-stomp-1.3.5-1.el7.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: rubygem-stomp and rubygem-stomp-doc are not at that URL

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 14]
```diff
@@ -202,55 +190,73 @@ rubygem-rake:
...
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm
-simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
-simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm
+simp-vendored-r10k:
+  :rpm_name: simp-vendored-r10k-2.6.2-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-2.6.2-1.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Seems inconsistent that the simp-vencored-r10k* packages are from packagecloud for el7, but download.simp-project.com here.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 15]
```diff
@@ -202,55 +190,73 @@ rubygem-rake:
...
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-r10k-2.6.2-1.noarch.rpm
+simp-vendored-r10k-gem-semantic_puppet:
+  :rpm_name: simp-vendored-r10k-gem-semantic_puppet-1.0.2-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-semantic_puppet-1.0.2-0.noarch.rpm
+simp-vendored-r10k-gem-text:
+  :rpm_name: simp-vendored-r10k-gem-text-1.3.1-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-text-1.3.1-0.noarch.rpm
 sudosh2:
   :rpm_name: sudosh2-1.0.2-2el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/sudosh2-1.0.2-2el7.x86_64.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: syslinux-tftpboot: another for CentOS vault

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml (hunk 1 above)
Comment (brandonrdn) [reply build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: I was updating to latest, must've missed this one. All puppet-agent packages are now at 1.10.14-1

File: build/distributions/CentOS/6/x86_64/yum_data/packages.yaml (hunk 2 above)
Comment (brandonrdn) [reply build/distributions/CentOS/6/x86_64/yum_data/packages.yaml]: python-six was only listed in CentOS 6 packages.yaml. Assuming it was removed, just not from this one. Removed it and will re-add if any issues arise.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 7 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Fixed CentOS7 link. Tested 6 and the link worked fine.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 8 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Updated

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 9 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Was this for gweb? If so, the URL has been updated. Had to add /download.rpm to the URL. The other 4 in URLs listed in the code block worked.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 10 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Fixed

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 13 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Looks like earlier versions than what we had are on the epel mirrors now. Also, none are listed for el7, only el6. Should we remove them?

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 14 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Fixed

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml (hunk 15 above)
Comment (brandonrdn) [reply build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Fixed. Needed for el6?

**Comment by lnemsick-simp** [comment]
By manually downloading RPMs that failed during the build, I was able to build CentOS6 and CentOS7 ISOs.

**Pull request merged**